CUDADataFormats/Track/interface/TrajectoryStateSoAT.h

```diff
@@ -25,12 +25,12 @@ struct TrajectoryStateSoAT {
     auto cov = covariance(i);
     cov(0) = ccov(0, 0);
     cov(1) = ccov(0, 1);
-    cov(2) = b * float(ccov(0, 2));
+    cov(2) = b * static_cast<float>(ccov(0, 2));
```

Comment:
Fine by me - but I think that `float(...)` is a constructor, not a C-style cast.